src/config/db_version

```diff
@@ -1 +1 @@
-7f67e66
\ No newline at end of file
+i463
```

Comment:
Once you have merge in the database PR this should be updated to point at the new hash of db master.